kobo_highlights/main.py

```diff
@@ -225,6 +225,7 @@ def show_config():
     """Show the current program configuration."""
     global config
     console.print(Panel(config, expand=False))
+    console.print("[yellow]Configuration file path:", CONFIG_PATH)
```

Comment:
I like the idea of showing there the config file is, though I'd rather shot it as a panel title; I think it's harder to miss it this way.

```suggestion
    console.print(
        Panel(
            config,
            expand=False,
            title=str(CONFIG_PATH),
            padding=(1, 0, 0, 0),
        )
    )

```